src/client/ContentManagement.js

```diff
@@ -1530,3 +1529,70 @@ exports.SetAuthPolicy = async function({objectId, policyId}) {
     metadata: { "elv:delegation-id": policyId }
   });
 };
+
+/**
+ * Set the tenant contract ID for the libraryAddress provided
+ *
+ * @param libraryAddress
+ * @param tenantContractId
+ * @returns {Promise<void>}
+ */
+exports.SetLibraryTenantContractId = async function({libraryAddress, tenantContractId}){
```

Comment:
I don't think we need individual methods for every type. You should be able to do one method for Get and one for Set, taking an address or a id and looking up the library ID (if necessary) with client.ContentObjectLibraryId

Also the general format of methods in the client is that accessors are just the name of the thing to be accessed (i.e. ContentObjectTenantContractId instead of GetContentObjectTenantContractId)